ServiceWebsite/ServiceWebsite.Services/HearingsService.cs

```diff
@@ -43,7 +43,34 @@ private static bool HearingContainsParticipant(HearingDetailsResponse response,
         {
             return response.Participants.Any(p => p.Username.Equals(username, StringComparison.OrdinalIgnoreCase));
         }
-        
+
+        public async Task<Guid?> GetParticipantIdAsync(string username, Guid hearingId)
+        {
+            try
+            {
+                var hearingResponse = await _bookingsApiClient.GetHearingDetailsByIdAsync(hearingId);
+                var participant = hearingResponse.Participants.FirstOrDefault(p => p.Username.Equals(username, StringComparison.OrdinalIgnoreCase));
+
+                if(participant==null)
+                {
+                    return Guid.Empty;
```

Comment:
this should return a `Nullable<Guid>()` without value

Reply:
Resolved